### DISCUSSION TITLE
I want to cancel my account

### DISCUSSION THREAD

**Discussion by younsang2000** [community/community#69222] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I want to cancel my account

**Comment by AvinashAnand02** [comment]
Cancle account ? you want to say deactivate ? or someting else please elobrate your problem

**Comment by younsang2000** [comment]
I want to cancel my account and deactivate my account as soon as possible
Thanks

2023년 10월 4일 (수) 오후 1:48, Avinash Anand ***@***.***>님이 작성:

> Cancle account ? you want to say deactivate ? or someting else please
> elobrate your problem
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/69222#discussioncomment-7190941>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BCPL3S7KOI33POXKKJRYUQDX5W4R3AVCNFSM6AAAAAA5S3O2JWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TCOJQHE2DC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
김윤상 보나벤투라
Youn Kim
General Manager
Big Lake Inn
PO Box 456
112-12 Avenue SW
Slave Lake, AB, T0G2A0

P.780-849-6668
F.780-849-6678

***@***.***

*Copyright © 2015 Big Lake Inn*
You are receiving this email because you are staying at Big Lake Inn and/or
provided your address. To ensure future delivery please add
***@***.*** ***@***.***> to your contacts or safe list.
If you are not the intended recipient, you must not disclose or use the
information contained herein. If you have received this email in error,
please notify us immediately by return email and delete this message and
any attachments.

**Comment by radar-blip** [comment]
@younsang2000 You can go into settings, and then to Account (in left column) 
and there you will see how to delete your account.

**Comment by MohammedMSalha** [comment]
Profile ---> Delete your account
